homeassistant/components/modbus/base_platform.py

```diff
@@ -92,8 +120,6 @@ def __init__(self, hub: ModbusHub, entry: dict[str, Any]) -> None:
         self._attr_device_class = entry.get(CONF_DEVICE_CLASS)
         self._attr_available = True
         self._attr_unit_of_measurement = None
-        self._lazy_error_count = entry[CONF_LAZY_ERROR]
-        self._lazy_errors = self._lazy_error_count
```

Comment:
If this this planned for 2024.7, the what is changed here, and on other affected and removed code?

Reply:
For now we leave the parameter so that people have time to adapt configuration.yaml, but the function is of the parameter is changed in this PR.

In 2024.7 the parameter will be removed, and thus if present in configuration.yaml it will be invalid.

This is actually how I have done it with other parameters....make the change now, but continue to accept the parameter, while creating an issue. That way the user is warned, that the functionality have changed, and he has a number of month to correct configuration.yaml

Reply:
I understand, so it is a breaking change, you are not deprecating, but removing functionality. It is not going to break/stop working with 2024.7.x, it will break with 2024.1.x if this gets released.
May be you remove the issue and logging by 2024.7.x but that can just be added as a code comment.

Reply:
But the functionality will break?

Reply:
No actually it will not. The keyword was used to suppress X identical errors before starting to log them, this is no longer needed.

Reply:
Please add this in the breaking change section as it is a change of behavior.

Reply:
Copy/pasted your text.

Reply:
Thnx